tests/e2e/utils/utils.py

```diff
@@ -85,6 +85,21 @@ def wait_for(callback, timeout=300, interval=30):
             time.sleep(interval)
 
 
+def wait_for_kfp_run_succeeded_from_run_id(kfp_client, run_id):
```

Comment:
+1 now, I feel we should start aws_utils, kfp_utils, kube_utils at some point

Reply:
yea that sounds good.